serverless/deploy/github/sequence_and_integrate/entrypoint.sh

```diff
@@ -4,34 +4,40 @@ set -e
 
 function main {
     if [ "${SERVERLESS_LOG_PRIVATE_KEY}" == "" ]; then
-    echo "Missing log private key."
-    exit 1
+        echo "Missing log private key."
+        exit 1
     fi
     if [ "${INPUT_LOG_DIR}" == "" ]; then
-    echo "Missing log dir input."
-    exit 1
+        echo "Missing log dir input."
+        exit 1
     fi
     echo "::debug:Log directory is ${GITHUB_WORKSPACE}/${INPUT_LOG_DIR}"
 
     cd ${GITHUB_WORKSPACE}
 
-    PENDING="${INPUT_LOG_DIR}/leaves/pending"
-    if [ ! -d "${PENDING}" ]; then
-        echo "::debug:Nothing to do :("
-        exit
+    ECOSYSTEM=""
+    if [ "${INPUT_ECOSYSTEM}" == "" ]; then
```

Comment:
`!=` ?

Reply:
Right..
In the last commit I also removed the `ECOSYSTEM` variable entirely as otherwise the double quotes are always included in the ecosystem string.

Reply:
I'd rather avoid the duplicated invocations if possible - more chance of them getting out of sync (and multiplying if another optional flag comes along).

Does this work:

```bash
ECOSYSTEM_FLAG=""
if [ ! -z "${INPUT_ECOSYSTEM}" ]; then
  ECOSYSTEM_FLAG=--ecosystem="${ECOSYSTEM}"
fi

...

/bin/integrate ... "${ECOSYSTEM_FLAG}"
```

?

Reply:
Unfortunately this does not work when INPUT_ECOSYSTEM is empty.
This is what you get with set -x:

```
INPUT_ECOSYSTEM="foobar" ./entrypoint.sh
+ echo '::debug:No checkpoint file - initialising log'
::debug:No checkpoint file - initialising log
+ /tmp/integrate --storage_dir=log --ecosystem=foobar --initialise --logtostderr
+ exit

INPUT_ECOSYSTEM="foo bar" ./entrypoint.sh
+ echo '::debug:No checkpoint file - initialising log'
::debug:No checkpoint file - initialising log
+ /tmp/integrate --storage_dir=log '--ecosystem=foo bar' --initialise --logtostderr
+ exit

./entrypoint.sh
+ echo '::debug:No checkpoint file - initialising log'
::debug:No checkpoint file - initialising log
+ /tmp/integrate --storage_dir=log ' ' --initialise --logtostderr
F0707 12:39:30.462229   73538 main.go:99] Failed to read log checkpoint: "open log/checkpoint: no such file or directory"

Reply:
Much sad :/ 
Ok let's run with the duplicates for now to unblock you and figure it out a bit later.